res/assets/health.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<!DOCTYPE assetlist>
+
+<assetlist name="Health">
+    <!-- Note: Doctor Healing animation frames 01-06 -->
+    <image id="Doctor heal 01" src="Doctor_heal_plague_01"/>
```

Comment:
Note: since in the game code you only actively call the first frame (the rest are obtained from an offset counting from the first), you don't really need to define an id to any image except the first.